sosw/worker.py

```diff
@@ -61,23 +61,23 @@ class Worker(Processor):
     lambda_client = None
 
 
-    def __call__(self, event: Dict):
+    def __call__(self, event: Dict, result=None):
```

Comment:
That is a very bad idea to change the signature of the `__call__` method.
We can create a `self.result` for example and document this, but result has nothing to do with the `__call__` input.